customTests/azure_cpu_stream.nhc

```diff
@@ -2,7 +2,26 @@
 
 # Run Stream
 
+function get_core_conf(){
+    # only supports HBv4/HX
+    local max_cores=$(nproc)
 
+    if [[ $max_cores != 176 ]]; then
+        local increment=$((max_cores / 24))
+        local sequence=""
+
+        for ((i = 0; i < max_cores; i += increment)); do
+            sequence+="{$i:1},"
+        done
+
+        # Remove the trailing comma
+        sequence=${sequence%,}
+
+        echo "$sequence"
+    else
+        echo '{0:1},{8:1},{16:1},{24:1},{32:1},{38:1},{44:1},{52:1},{60:1},{68:1},{76:1},{82:1},{88:1},{96:1},{104:1},{112:1},{120:1},{126:1},{132:1},{140:1},{148:1},{156:1},{164:1},{170:1}'
```

Comment:
For the particular size you are testing is 24 cores w/ mavx512f still performing better than 48 cores w/ mavx2?

Reply:
addressed